### DISCUSSION TITLE
Add error bars in plot_bar

### DISCUSSION THREAD

**Discussion by eperezv** [ChiLiubio/microeco#186] [Q&A]
Hello,
Thank you for coding this wonderful package, I find it really useful.
I have a question. Is it possible to add error bars to plots generated by plot_bar? 
Thank you,
Eduardo

**Comment by ChiLiubio** [comment]
Hi,
Thanks. Do you mean adding error bars in the stacked bar plot from plot_bar function of trans_abund class? It has not been tried in the previous development. The main reason is it is hard to make the figure clear with multiple error bars in the stacked bar plot. Anyway, the feature will be considered in the future release.

Chi

**Reply by eperezv** [reply]
Yes, like the ones here: https://stackoverflow.com/questions/30872977/how-to-stack-error-bars-in-a-stacked-bar-plot-using-geom-errorbar
I understand that multiple error bars in a stacked bar plot can be messy (especially with high number of taxa) but I wanted to give it a try. 
Thanks for your quick answer
